Update readmes to match common design (KML category) 2/2 (#460)
Esri/arcgis-runtime-samples-java: kml/play-a-kml-tour/README.md

# Play KML Tour

Play tours in KML files.

![Image of play KML tour](PlayAKMLTour.png)

## Use case

KML, the file format used by Google Earth, supports creating tours, which can control the viewpoint of the scene, hide and show content, and play audio. Tours allow you to easily share tours of geographic locations, which can be augmented with rich multimedia. Runtime allows you to consume these tours using a simple API.

## How to use the sample

> The sample will load the KMZ file from ArcGIS Online. When a tour is found, the _Play_ button will be enabled. Use _Play_ and _Pause_ to control the tour. When you're ready to show the tour, use the reset button to return the tour to the unplayed state.

Reviewer: we need to tweak this for our sample, though I've noticed a UI ?bug/behaviour difference. To replay, first we have to hit refresh, and then play.

```suggestion
The sample will load the KMZ file from ArcGIS Online. Click the play button to start the tour. The narration audio will start and then the viewpoint will animate. Press the button again to pause the tour. To restart the tour, hit the refresh button and then the play button.
```

## How it works

> 1. Load the KML file and add it to a layer.

Reviewer: ```suggestion
1. Create and load a `KmlLayer` with a `KmlDataset` and add it to the scene's operational layers.
```

> 2. Create the KML tour controller. Wire up the buttons to the `Play`, `Pause`, and `Reset` methods.

Reviewer: We should remove the UI references here since they don't match Java anyway and we should refrain from UI wiring up in this section
```suggestion
2. Create a `KMLTourController` and set the tour with `kmlTourController.setTour(kmlTour)`.
```

> 3. Explore the tree of KML content to find the first KML tour. Once a tour is found, provide it to the KML tour controller.

Reviewer: Since this isn't the order we do this in Java, I think we should use what we had previously for this section. It doesn't differ from the design fully and it's more important that our readme match how we write the sample in Java. 

I'd recommend the previous version for ordering, we could re-word it to match more closely the words used in the common design though if you felt it deviated too much. We just need to ensure the ordering is correct (e.g. the recursing through Kmlnode step should come before the above mention of setting up the KmlTourController)

Author: No worries, happy to revert to the old descriptions and ordering. The contents are quite similar.

> 4. Enable the buttons to allow the user to play, pause, and reset the tour.

Reviewer: ```suggestion
4. Use `kmltourController.play()`, `kmltourController.pause()`, and `kmltourController.reset()` to control the tour.
```
